src/screens/BeerOfTheDay/BeerOfTheDay.jsx

```diff
@@ -7,58 +7,60 @@ import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteBu
 import Button from '../../shared/components/Button/Button';
 import './styles.scss';
 
-class BeerOfTheDayComponent extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      beer: [],
-    };
-  };
 
-  getRandomBeer = async () => {
+const createEmptyBeer = () => ({
+  name: '',
+  abv: '',
+  img: '',
+  ibu: '',
+  isPlaceholder: true,
+})
+
+function BeerOfTheDayComponent() {
+
+  const [beer, setBeer] = useState(createEmptyBeer())
+
+  const getRandomBeer = async () => {
```

Comment:
I am not a fan of naming functions with const and arrow function syntax. I strongly suggest to use function naming.

Reply:
Done!